integration/backfill_test.go

```diff
@@ -104,15 +96,15 @@ overrides:
 		})
 
 		// Try to upload block using mimirtool. Should fail because upload is not enabled for user.
-		output, err := runMimirtoolBackfill(t, "--address", compactorURL, "--id", "anonymous", block1Path)
+		output, err := runMimirtoolBackfill(tmpDir, compactor, block1)
 		require.Contains(t, output, fmt.Sprintf("msg=\"block uploaded successfully\" block=%s", block1))
 		require.NoError(t, err)
 	}
 
 	{
 		// Upload block1 and block2. Block 1 already exists, but block2 should be uploaded without problem.
-		output, err := runMimirtoolBackfill(t, "--address", compactorURL, "--id", "anonymous", block1Path, block2Path)
-		require.Contains(t, output, fmt.Sprintf("msg=\"block already exists on the server\" path=%s", block1Path))
+		output, err := runMimirtoolBackfill(tmpDir, compactor, block1, block2)
+		require.Contains(t, output, fmt.Sprintf("msg=\"block already exists on the server\" path=%s", path.Join(e2e.ContainerSharedDir, block1.String())))
```

Comment:
[nit] Doesn't change anything when running on *nix.
```suggestion
		require.Contains(t, output, fmt.Sprintf("msg=\"block already exists on the server\" path=%s", filepath.Join(e2e.ContainerSharedDir, block1.String())))
```

Reply:
I used `filepath` for local paths (windows, mac or linux), and `path` for paths in Docker (linux). Since this message comes from mimirtool in docker, I used `path` here. You're right that it makes no difference when running integration test  on linux.